### PR TITLE
Improvement: add schema for profiling plugin options, remove todo item

### PR DESCRIPTION
**Pull request by EugeneHlushko**
**What kind of change does this PR introduce?**
* Adding schema for `ProfilingPlugin`
* Removed unused schema (belongs to deprecated plugin)
* Renamed parameter for readability
* Added documentation section

**Did you add tests for your changes?**
No

**If relevant, link to documentation update:**
webpack/webpack.js.org#1830

**Summary**
Was working on another change when bumped at this `TODO` item

**Does this PR introduce a breaking change?**
Yes, option was renamed

**Other information**

### CODE REVIEW

File: schemas/plugins/debug/ProfilingPlugin.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "id": "ProfilingPlugin",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "outPath": {
+      "description": "Path to the output file e.g. `profiling/events.json`. Defaults to `events.json`.",
+      "type": "string",
+      "absolutePath": true,
```
[the remark this answers is not in the document]
Comment (EugeneHlushko) [reply schemas/plugins/debug/ProfilingPlugin.json]: hmm i thought i'd set this to false, it does indicate it as path (absolute/relative) and currently set to abs... will set to false. validation uses ajv.absolutePath.js
https://github.com/webpack/webpack/blob/33c907e1b76bc7902746b9b10cd0683a448fc67e/schemas/ajv.absolutePath.js#L15-L29

**Comment by EugeneHlushko** [comment]
The fail is redundant, test was updated

**Comment by sokra** [comment]
Thanks

**Comment by EugeneHlushko** [comment]
<3

**Pull request merged**